doc/problem/overview.md

```diff
@@ -1,26 +1,71 @@
+
+This project adopts Lorenz?s approach--observing the whole spectrum of animal behavior in their natural state. Previous approaches relied on human observation to score animal behavior. However, slow data collection, low dimensional data, imprecise and subjective measurements, and human visual and language limitations impeded data collection and analysis. Using modern quantitative tools for measurement, description, and analysis, the field of [computational ethology](#computational-ethology1) has emerged to solve these issues. Together, modern mathematics, engineering, and computer science have the potential to establish a causal relationship between genetics and behavior.  
+
+<div id='why-mice'/>
+### 1.3 Why Mice? 
+
+Although the human genome was first sequenced over a decade ago, the relationship between genetics and behavior is still not well understood. For ethical reasons, genes cannot be systematically manipulated in humans. Therefore, the familiar scientific testing approach must be edited -- enter mice. There are several reasons the mouse has become the mammal of choice in human behavioral studies. Chief among them is the fact that "approximately 99% of mouse genes have human counterparts--conversely, mouse versions (orthologs) can be identified for 99% of human genes." Additionally, the brain organization and behavioral responses of humans and mice display many similarities. For example, both mammals display complex processes like hunger and fear. This allows scientists to more easily identify and track these behaviors. In addition, the extreme similarities of intra-strain mice, the space efficiency of maintaining their caged environments, and the speed of reproduction make mice a logical alternative to testing on humans. 
+
+<div id='methodology-description'/>
+## 2. Methodology Description
+
+The traditional technology to analyze behavior is time-intensive and labor-intensive. For example, recording data requires researchers to track behavior uninterruptedly. This experiment utilized a method for continuous high-resolution behavioral data collection and analysis. The [home cage monitoring (HCM) system](#hcm-system) utilized is a network of photobeam feeding detection, drinking detection, and activity platform sensors that records mouse [active states](#active-state) and [inactive states](#inactive-state) automatically and incessantly over 24-hour periods. This facilitates objective, multi-dimensional computer tracking, providing a higher degree of accuracy compared to human observation.
+
```

Comment:
I think they said it was over 23 hours of the day with 1 hour for cleaning where they didn't track sensor data?